downstream/modules/platform/proc-choosing-obtaining-installer.adoc

```diff
@@ -34,13 +34,13 @@ $ tar xvzf ansible-automation-platform-setup-<latest-version>.tar.gz
 v.2.2 for RHEL 8 for x86_64
```

Comment:
Should this be changed to 'v.2.3 for RHEL 8'?

Reply:
Not sure about that. I wonder if there's a corresponding version for RHEL versions. I'll check.